drivers/hmis_external_apis/lib/tasks/ac_hmis.rake

```diff
@@ -0,0 +1,10 @@
+namespace :ac_hmis do
+  task :update_unit_availability, [:force] => :environment do |_task, args|
+    data_source = HmisExternalApis::AcHmis.data_source
```

Comment:
we should return unless `HmisEnforcement.hmis_enabled?`

Reply:
Good catch, will fix. Thanks!